public/app/plugins/datasource/elasticsearch/components/MetricPicker.tsx

```diff
@@ -23,12 +23,16 @@ interface Props {
   value?: string;
 }
 
-export const MetricPicker: FunctionComponent<Props> = ({ options, onChange, className, value }) => (
-  <Segment
-    className={cx(className, noWrap)}
-    options={toOptions(options)}
-    onChange={onChange}
-    placeholder="Select Metric"
-    value={!!value ? toOption(options.find((option) => option.id === value)!) : null}
-  />
-);
+export const MetricPicker: FunctionComponent<Props> = ({ options, onChange, className, value }) => {
+  const selectedOption = options.find((option) => option.id === value);
```

Comment:
Unrelated change, but this was breaking in certain situations when expanding the options for metrics with no fields